test/integration/core/exceptions.tap.js

```diff
@@ -10,6 +10,9 @@ const cp = require('child_process')
 const path = require('path')
 const helper = require('../../lib/agent_helper')
 const helpersDir = path.join(path.resolve(__dirname, '../../'), 'helpers')
+const fs = require('fs')
+const pipePath = '/tmp/newrelic-telemetry'
+const shouldUsePipe = fs.existsSync(pipePath)
```

Comment:
Introduce variable paths into the test seems dangerous and likely to increase chance of test flickers or confusion trying to debug locally VS CI, etc.

We should be able to guarantee one state or the other for the sake of the test. Perhaps we need two tests here to exercise both paths? Or do we really only need to execute one style?

Reply:
Ultimately, I'm happy to call this an artifact of my local testing.  I was running into issues with the exceptions test failing when the pipe was present, but a refactoring has fixed that, so...I don't think a change to exceptions is necessary.